tests/affiliatedcertification/affiliatedcerthelper/operator.go

```diff
@@ -106,3 +108,67 @@ func IsOperatorInstalled(namespace string, operatorDeploymentName string) error
 
 	return nil
 }
+
+func DeployRHCertifiedOperatorSource(ocpVersion string) error {
+	err := globalhelper.APIClient.Create(context.TODO(),
+		&v1alpha1.CatalogSource{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "certified-operators",
+				Namespace: "openshift-marketplace",
+			},
+			Spec: v1alpha1.CatalogSourceSpec{
+				SourceType:  "grpc",
+				Image:       "registry.redhat.io/redhat/certified-operator-index:v" + ocpVersion,
+				DisplayName: "redhat-certified",
+				Publisher:   "Redhat",
+				Secrets:     []string{"redhat-registry-secret", "redhat-connect-registry-secret"},
+				UpdateStrategy: &v1alpha1.UpdateStrategy{
+					RegistryPoll: &v1alpha1.RegistryPoll{
+						Interval: &metav1.Duration{
+							Duration: 30 * time.Minute}},
+				},
+			},
+		},
+	)
+	if err != nil {
+		return fmt.Errorf("can not deploy catalog source %w", err)
+	}
+
+	return nil
+}
+
+func DisableCatalogSource(name string) error {
```

Comment:
This function looks the same as EnableCatalogSource.
You can create common private function setCatalogSource(enable bool, name string) and re-use this helper function from DisableCatalogSource and EnableCatalogSource